rclcpp_examples/src/topics/listener_memory.cpp

```diff
@@ -56,5 +58,7 @@ int main(int argc, char * argv[])
 
   executor.spin();
 
+  printf("Received %d messages.\n", messages_received);
```

Comment:
You can also use `std::to_string` with `std::cout` instead of `printf` to make it more C++ idiomatic, but it's just a cosmetic change.